lib/chef/provider/service/systemd.rb

```diff
@@ -173,25 +198,29 @@ def unmask_service
   end
 
   def is_active?
-    options, args = get_systemctl_options_args
-    shell_out(systemctl_path, args, "is-active", new_resource.service_name, "--quiet", **options).exitstatus == 0
+    status = systemd_service_status
+    # Note: "activating" is not active (as with type=notify or oneshots)
```

Comment:
```suggestion
    # Note: "activating" is not active (as with type=notify or one-offs)
```

`oneshots` is not a dictionary word so this is failing our spellchecker.

Reply:
'oneshot' is specific systemd terminology.  Is there an incantation to tell the spellchecker to back off?

Reply:
Ahh, then in that case you'll need to add it to the list of exceptions in the `cspell.json` file: https://github.com/chef/chef/blob/master/cspell.json#L16